qds_sdk/commands.py

```diff
@@ -341,25 +343,38 @@ def validate_program(cls, options):
     @classmethod
     def validate_cmdline(cls, options):
         bool_cmdline = options.cmdline is not None
-        bool_other_options = options.script_location is not None or options.program is not None
+        bool_other_options = options.script_location is not None or options.program is not None or options.sql is not None
 
         # if both are false then no option is specified ==> raise ParseError
         # if both are true then atleast two option specified ==> raise ParseError
         if bool_cmdline == bool_other_options:
-            raise ParseError("Exactly One of script location or program or cmdline should be specified", cls.optparser.format_help())
+            raise ParseError("Exactly One of script location or program or cmdline or sql should be specified", cls.optparser.format_help())
         if bool_cmdline:
             if options.language is not None:
                 raise ParseError("Language cannot be specified with the commandline option", cls.optparser.format_help())
 
     @classmethod
+    def validate_sql(cls, options):
+        bool_sql = options.sql is not None
+        bool_other_options = options.script_location is not None or options.program is not None or options.cmdline is not None
+
+        # if both are false then no option is specified => raise PraseError
+        # if both are true then atleast two option specified => raise ParseError
+        if bool_sql == bool_other_options:
+            raise ParseError("Exactly One of script location or program or cmdline or sql should be specified", cls.optparser.format_help())
+        if bool_sql:
+            if options.language is None:
+                raise ParseError("Unspecified language for Sql query", cls.optparser.format_help())
```

Comment:
I have tried to send all the parameters which we are sending from the Analyze UI and language is one of them. I will remove 'language' and run the query again and see if it succeeds.